### PR TITLE
Php and navbar update

### PR DESCRIPTION
**Pull request by rbaia28**
updated form php to what we covered in class today
changed name "tag" to say full name for clarity 
linked nav bar to return to index page when selecting "Pizza Party" in top corner 
removed extra PHP page from index and combined the files to one document 
changed contact email to Gagan's non-gmail account because gmail has stupid filters that stop these forms apparently
addressed ticket #23

### CODE REVIEW

File: contactus.php [hunk 1]
```diff
@@ -42,17 +47,60 @@
...
+        $subject = '';
+        $message = '';
+        $mailFrom = '';
+
+    if (isset($_Post['submit-btn'])) {
+
+         if (!empty(trim($_POST['name']))) {
+            $fullname = trim($_POST['name']);
+        } else {
+            echo "<p> Name cannot be blank</p>";
```
Comment (billythakidd04) [inline contactus.php]: Consider setting an error variable here and then using that error to display something on or near the input in question.

for example:
on line 63 - `$error['fullname'] = true; //or some message...`

then at the input you could do:
`<input class="/*all the regular stuff*/ <?=isset($error['fullname']?'error':''?>"....` and maybe use a bootstrap error class for form elements...

File: contactus.php [hunk 2]
```diff
@@ -42,17 +47,60 @@
...
         <form class="contact-form" action="contactus.php" method="post">
-            <input type="text" name="name" placeholder="Full Name">
-            <input type="text" name="mail" placeholder="Your Email">
-            <input type="text" name="subject" placeholder="Subject">
-            <textarea name="message" placeholder="Message"></textarea>
-            <button type="submit" name="submit">Send Mail</button>
+            <input type="text" name="full name" <?= ($fullname ? 'value="' . $fullname . '"' :  ''); ?> placeholder="Full Name"required><br/>
+            <input type="email" name="mail" <?= ($email ? 'value="' . $email . '"' : ''); ?> placeholder="Your Email" required><br/>
+            <input type="text" names="subject" <?=($subject ? 'value="' . $subject . '"' : ''); ?> placeholder="Subject" required></br/>
+            <textarea name="message" placeholder="Tell us about your experience! Better yet..dont." required><?= ($message ? $message : ''); ?></textarea><br/>
```
Comment (billythakidd04) [inline contactus.php]: LOL! what is that placeholder?! XD

File: contactus.php [hunk 3]
```diff
@@ -42,17 +47,60 @@
...
         <h1> Connect with Us! </h1></br>
-        <p>Send E-mail</p>
+        <p>Send E-mail</p><br/>
         <form class="contact-form" action="contactus.php" method="post">
-            <input type="text" name="name" placeholder="Full Name">
-            <input type="text" name="mail" placeholder="Your Email">
-            <input type="text" name="subject" placeholder="Subject">
-            <textarea name="message" placeholder="Message"></textarea>
-            <button type="submit" name="submit">Send Mail</button>
+            <input type="text" name="full name" <?= ($fullname ? 'value="' . $fullname . '"' :  ''); ?> placeholder="Full Name"required><br/>
```
Comment (billythakidd04) [inline contactus.php]: `full name` is not the value you want here...

File: contactus.php [hunk 4]
```diff
@@ -42,17 +47,60 @@
...
+            <input type="email" name="mail" <?= ($email ? 'value="' . $email . '"' : ''); ?> placeholder="Your Email" required><br/>
+            <input type="text" names="subject" <?=($subject ? 'value="' . $subject . '"' : ''); ?> placeholder="Subject" required></br/>
+            <textarea name="message" placeholder="Tell us about your experience! Better yet..dont." required><?= ($message ? $message : ''); ?></textarea><br/>
+            <input type="submit" name="submit-btn"></br>
         </form>
-        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+    </div>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+<--add checkbox for subscribe for coups
```
Comment (billythakidd04) [inline contactus.php]: Whats going on here?

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Few questions, one definite bug, and a suggestion

File: contactus.php (hunk 2 above)
Comment (rbaia28) [reply contactus.php]: :D just keeping things exciting lol. Updated in new copy.

File: contactus.php (hunk 4 above)
Comment (rbaia28) [reply contactus.php]: forgot to use the proper commenting shortcut. Was for the group to potentially discuss including. Has been removed

File: contactus.php (hunk 1 above)
Comment (rbaia28) [reply contactus.php]: This has been seen and is being worked through as a team. updates incoming.

**Review by billythakidd04 — APPROVED** [review]

**Pull request merged**